eo2js/test/commands/transpile.test.js

```diff
@@ -49,6 +49,25 @@ describe('transpile', function() {
       fs.writeFileSync(path.resolve(target, 'eo-foreign.json'), JSON.stringify(foreign))
       fs.mkdirSync(path.resolve(target, '6-verify/com/eo2js'), {recursive: true})
       fs.copyFileSync(path.resolve(`test/resources/transpile/${name}.xmir`), verified)
+    }
+    /**
+     * Run transpile command with verbose output.
+     * @param {String} name - Name of the object to transpile, defaults to 'simple'
+     * @return {String} - Command stdout from transpilation
+     */
+    const transpile = function(name = 'simple') {
+      prepare(name)
+      return runSync([
```

Comment:
I think calling `retranspile` in `transpile` may result in a code smell because of two reasons:
- As an abstraction, re-something should call something not vice-versa
- What if we need to add more logic regarding retranspilation here in the future? This would cause a problem as those two functions are tightly coupled.

Because of the above, I decided to duplicate the code. What do you think?

One solution I thought of to avoid code duplication: 
- Remove`retranspile` and Modify the transpile function to accept the prepare boolean flag here. So, when we re-transpile we don't resume to the initial state (copying files on Windows gives a carbon copy of the source file. I mean the same mtime -> resulting test failing)

@maxonfjvipon